#!/usr/bin/env python
PACKAGE = "terrain_nav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("scale", double_t, 0, "Scale depth image size", 100.0)
gen.add("minX", double_t, 0, "MinX_L", -0.8)
gen.add("maxX", double_t, 0, "MaxX_L", 0.8)
gen.add("minY", double_t, 0, "MinY_L", -0.5)
gen.add("maxY", double_t, 0, "MinY_L", 0.15)
gen.add("minZ", double_t, 0, "MinZ_L", -2.5)
gen.add("maxZ", double_t, 0, "MinZ_L", +2.5)
gen.add("max_distance", double_t, 0, "Max distance", +1.0)
gen.add("limitMin", double_t, 0, "LimitMin", -0.1)
gen.add("limitMax", double_t, 0, "LimitMax", +0.1)
gen.add("radius", double_t, 0, "Radius", +0.03)
gen.add("which", int_t, 0, "which", 0)

exit(gen.generate(PACKAGE, "navParams", "navParams"))